src/index.js

```diff
@@ -1,8 +1,12 @@
+require('dotenv').config();
+const connectDB = require('./db');
 const app = require('./app');
 
 const port = process.env.PORT || 5050;
-app.listen(port, () => {
-  /* eslint-disable no-console */
-  console.log(`Listening: http://localhost:${port}`);
-  /* eslint-enable no-console */
+
+// Connect to MongoDB first
+connectDB().then(() => {  
```

Comment:
Same thing about this comment in here. 
Comments should not explain the code, if they do it means : 

Either that your code is not written good enough (it should be self explained) or the comment is redundent